modules/transitive/tests/src/ClosureSpec.scala

```diff
@@ -212,4 +212,48 @@ class ClosureSpec extends munit.FunSuite {
     )
     assert(result.isSuccess)
   }
+
+  test(
```

Comment:
these are 2 kinds of cycles situations I can think of:
- recursive definition
- cyclic reference through traits (this is what made us discover this issue)